packages/user-service/.env.example

```diff
@@ -14,8 +14,12 @@ DB_NAME=db_name
 # DB_USER=
 # DB_PASSWORD=
 
+## Certificate Configuration
+SSL_KEY=/etc/pki/tls/private/server.key
+SSL_CERT=/etc/pki/tls/certs/server.crt
```

Comment:
You can leave these values empty, as this is just an example file.
This should go in the actual `.env` file.

Reply:
What the yeoman generator does is it copys the .env.example file to actual .env file on bootstraping

https://github.com/1-Platform/one-platform/blob/master/cli/generator-op-microservice/generators/microservice/index.js#L267

Reply:
Still the values should not be in the `.env.example`. The .env file is meant to be edited by the developers after creation.

Reply:
But some of them are basic things i suggest to keep the location there

Reply:
who ever uses it can be modified.

Reply:
We can make a note in the cli that default location is in `/etc` if you want to modify change it from there. i

Reply:
The default location is wherever they've created the certificates. Not necessarily in `/etc`.
@sayak-sarkar Can you check and see if this is okay?

Reply:
Instead of specifying in the ```/etc/pki/tls/``` path explicitly, I would rather put in just a comment asking to put in the actual location cert file as the cert files can exist anywhere within the system based upon the component requirements/container configuration/etc.

Reply:
Ok i have updated the configuration as suggested.